src/components/about/ContactCard.js

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import {
+  Flex,
+  Image,
+  Box,
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+  Text,
+  useTheme,
+  Link,
+} from '@chakra-ui/core';
+
+const CardContent = ({ title, blurb }) => {
+  const theme = useTheme();
+  return (
+    <Flex
+      w="100%"
+      direction="column"
+      p="4"
+      align="center"
+      justify="center"
+      textAlign="center"
+    >
+      <Text fontSize={theme.fontSizes.lg} fontFamily={theme.fonts.heading}>
+        {title}
+      </Text>
+      <Text
+        p="4"
+        fontSize={theme.fontSizes.paragraph}
+        fontFamily={theme.fonts.heading}
+      >
+        {blurb}
+      </Text>
+    </Flex>
+  );
+};
+
+// @TODO :: Add a fallback image to be displayed incase of no image
+const NoImage = () => (
+  <Box maxW="342px" maxH="220px" bg="#414A4C">
+    <Flex justify="center" align="center">
+      <p>No image available</p>
+    </Flex>
+  </Box>
+);
+
+// @TODO :: Replace with new Image component
+const CardImage = ({ imageUrl, imageAlt }) => (
+  <Flex w="342px" h="220px">
+    <Image width="100%" height="100%" src={imageUrl} alt={imageAlt} />
```

Comment:
Can we drop an `objectFit="cover"` property on here as well? Image is getting squished https://gyazo.com/d9ace5a93fb1a83462db728022ad1946

Reply:
Yep, good shout